### DISCUSSION TITLE
Artistic Palettes

### DISCUSSION THREAD

**Discussion by bryevdv** [bokeh/bokeh#11898] [General]
Recently a request for this "artistic" R palette package to be in "Python vis" came by on twitter:

* https://github.com/BlakeRMills/MetBrewer

There are also other very nice (to me) artistic palette repos:

* https://github.com/jakelawlor/PNWColors
* https://github.com/karthik/wesanderson

I have been interested in and experimenting with using Bokeh for generative art, so having palettes like these in a ready-to-consume format would be appreciated. By me at least, but I expect others might also be interested in them as well, for more "design-focused" usage. So the question is: where could they go? Is there any objection to a separate `bokeh/art-palettes` repository and package under the Bokeh org? I don't want to maintain a published package from my own personal repo (but I do want to publish this as a package).

I would not plan to mention these in the main docs beyond passing a sentence with a link to the package, and make it clear in the package docs that the palettes are derived from artistic considerations, not necessarily dataviz principles (athough FWIW the PNWColors claims to be evaluated with chroma.js for color-blind safety).

cc @bokeh/dev @karen-poon @jbednar

**Comment by p-himik** [comment]
> Is there any objection to a separate `bokeh/art-palettes` repository and package under the Bokeh org?

On the contrary - I think it's the way to go.

One small addendum - the palettes should be done in a way that does not rely on Bokeh at all. It should be just data that's obvious to use, with no classes. This way, it would be possible to use it from any Python code, maybe even from old Python versions - not with just the most recent Bokeh version running on top of the most recent Python version.

**Reply by bryevdv** [reply]
Yes, exactly, just tuples of rgba color values is all I had in mind.

**Comment by karen-poon** [comment]
I really like this idea!
